src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <>
+    <section className="Person">
+      <h2 className="Person__name">
+        {person.name}
+      </h2>
+      {person.age && (
+        <p className="Person__age">
+            {person.age}
+        </p>
+      )}
+      <p className="Person__partner">
+        {person.isMarried === true ? `${person.partnerName} is my  ${person.sex === 'f' ? 'wife' : 'husband'}` : 'I am not married'}
```

Comment:
Move this logic to a variable to keep JSX clean and organized. And it doesn't work correctly BTW. Natasha can't be a husband and Maksym can't be a wife 
![image](https://github.com/mate-academy/react_person/assets/99403198/48f0511b-a669-4cb4-9199-265c5fc75fff)